include/image-commands.mk

```diff
@@ -226,6 +226,11 @@ define Build/copy-file
 	cat "$(1)" > "$@"
 endef
 
+define Build/dlink-sge-image
+	$(STAGING_DIR_HOST)/bin/dlink-sge-image $(1) $@ $@.enc
+	mv $@.enc $@
```

Comment:
This was added in 4c0dc68f466588bb2ef60b29a4d8052b34112ca6, please rebase.

Reply:
Done